lib/bootstrap.js

```diff
@@ -0,0 +1,65 @@
+var spawn = require('child_process').spawn;
+var path = require('path');
+var sleep = require('sleep').sleep;
+
+function Bootstrap(codecept, callback) {
+
+  var phantomjs, seleniumPid, callbackCalled = false;
+
+  (function () {
+
+    var protractor = codecept.config.helpers.Protractor;
+    if (protractor && protractor.browser) {
+      if (/phantom/.test(protractor.browser)) {
```

Comment:
Sure, sure!! I'm agree with you. I think that a better approach is use a `ES6` class structure, or a Closure. I did this feature during one day, writing a behavior for a app in my job. I will make a refactor for that. Many thanks :)